media_driver/agnostic/gen10/vp/hal/vphal_renderer_g10.cpp

```diff
@@ -166,12 +166,18 @@ MOS_STATUS VphalRendererG10::AllocateRenderComponents(
 
 MOS_STATUS VphalRendererG10::InitKdllParam()
 {
-    MOS_STATUS eStatus = MOS_STATUS_SUCCESS;
+    MOS_STATUS eStatus = MOS_STATUS_UNIMPLEMENTED;
 
     // Set KDLL parameters (Platform dependent)
     pKernelDllRules         = g_KdllRuleTable_g10;
+#ifndef _FULL_OPEN_SOURCE
```

Comment:
hm. Question: should we cover:
`pKernelDllRules         = g_KdllRuleTable_g10;`
under #ifndef _FULL_OPEN_SOURCE as well?

@zliu50 : would this be ok? In this way, I think we can completely exclude a source file containing g_KdllRuleTable_g10 from the build if _FULL_OPEN_SOURCE is requested, thus avoid any modifications in it which I believe you did not like.